migrations/20160803083413-ibm-252-centre-model-prebooking-associations-removed.js

```diff
@@ -0,0 +1,51 @@
+"use strict";
+
+const cps = require('cps');
+
+exports.up = (db, callback) =>
+  cps.seq([
+    (_, callback) => db.removeColumn('centre', 'male_prebooking', callback),
+    (_, callback) => db.removeColumn('centre', 'female_prebooking', callback),
+    (_, callback) => db.removeColumn('centre', 'male_contingency', callback),
+    (_, callback) => db.removeColumn('centre', 'female_contingency', callback)
+  ], callback);
+
+exports.down = (db, callback) =>
+  cps.seq([
+    (_, callback) =>
+      db.addColumn(
+        'centre',
+        'male_prebooking',
+        {
```

Comment:
you don't need to specify this, just that its an `int` at the minute you're probably adding a string here